app/electron/main.ts

```diff
@@ -6,6 +6,7 @@ import { app, BrowserWindow, dialog, ipcMain, Menu, MenuItem, screen, shell } fr
 import { IpcMainEvent, MenuItemConstructorOptions } from 'electron/main';
 import log from 'electron-log';
 import find_process from 'find-process';
+import fixPath from 'fix-path';
```

Comment:
I was checking and this module + the dependencies from the same author and they have not been updated for 3 years.
I wonder if we should send them a patch for updating the dependencies, or just try to bring in the relevant parts of the module's logic.

Reply:
I saw that suggestion about the dependencies, but thought the module was 'done'. But yeah, good call out.

The dependencies could be reduced to none. We don't need to strip ansi just to get PATH and SHELL, and in 2024 executing processes is fine with node/electron APIs.